tensorflow_datasets/core/image_visualizer.py

```diff
@@ -0,0 +1,222 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+
+"""Visualizer for image datasets.
+"""
+
+from absl import logging
+
+from tensorflow_datasets.core import dataset_utils
+from tensorflow_datasets.core import features as features_lib
+from tensorflow_datasets.core import lazy_imports_lib
+
+class ImageVisualizer():
+  """Parent Class for support of Image based datasets"""
+
+  def _extract_keys(self, ds_info, feature_type, secondary_key=None):
```

Comment:
Please use functions instead of methods when you're not using `self`. This make the code easier to read (as it makes it clear for the reader what the input/outputs are (no hidden `self._xyz` inputs/outputs).